packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -0,0 +1,153 @@
+
+## End to end tests conventions
+
+**00-main-test-file.e2e.spec.js**
+```
+const E2eHelpers = require('platform/testing/e2e/helpers')
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const MyApplicationHelpers = require('./my-application-helpers.js');
+const testData = require('./schema/maximal-test.json');
+const FormsTestHelpers = require('platform/testing/e2e/form-helpers');
+
+// export the test using E2eHelpers.createE2eTest
+module.exports = E2eHelpers.createE2eTest(client => {
+  // mock api responses
+  MyApplicationHelpers.initApplicationSubmitMock();
+  MyApplicationHelpers.initEnrollmentStatusMock();
+
+  // use a comment to indicate what page is being tested
+  // Introduction page
+  client
+    .openUrl(`${E2eHelpers.baseUrl}/my-application`)
+    // use Timeouts constants
+    .waitForElementVisible('body', Timeouts.normal)
+    .assert.title('My Application Title | Veterans Affairs')
+    .click('.schemaform-start-button');
+
+  // disable scrolling
+  FormsTestHelpers.overrideFormsScrolling(client);
+
+  // assert navigation is successful
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+  // Personal Information page.
+  client.expect.element('input[name="root_veteranFullName_first"]').to.be
+    .visible;
+  // use functions from the helper file to perform all actions on the page
+  MyApplicationHelpers.completePersonalInformation(client, testData.data);
+  client.axeCheck('.main').click('.form-panel .usa-button-primary');
+  E2eHelpers.expectNavigateAwayFrom(
+    client,
+    '/my-application/personal-information',
+  );
+  ```
+
+**my-application-helpers.js**
+```
+const mock = require('platform/testing/e2e/mock-helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const Auth = require('platform/testing/e2e/auth.js');
+const moment = require('moment');
+
+function completePersonalInformation(client, data) {
+  client
+    .waitForElementVisible(
+      'input[name="root_veteranFullName_first"]',
+      Timeouts.normal,
+    )
+    .fill(
+      'input[name="root_veteranFullName_first"]',
+      data.veteranFullName.first,
+    )
+    .fill('input[name="root_veteranFullName_last"]', data.veteranFullName.last)
+}
+```
+_These are recommendations not requirements._
+
+- separate navigation from field input
+  - use a **main test file** for navigation, assertions, and calls helpers
+  - use a **helper file** that for filling out forms
+- create separate, numbered **main test files** to organize tests by their focus
+  - **00-all-fields.e2e.spec.js** - required and optional fields
+  - **01-required.e2e.spec.js** - only required fields
+  - **02-accessibility.e2e.spec.js** - validates accessibility
+  - **03-auth.e2e.spec.js** - validates authentication
+  - **04-cross-cutting-feature.e2e.spec.js** - validates one feature used across several pages (e.g. save in progress)
+- export the end to end test using `E2eHelpers.createE2eTest()`- see [Helpers](#helpers)
```

Comment:
not sure what's going on here- LMK if you have any ideas otherwise I'll dig into it. 
![Screen Shot 2019-05-20 at 12 03 40](https://user-images.githubusercontent.com/4998130/58042744-b47bc880-7af8-11e9-9c4e-50f2b055612a.png)

Reply:
what if you place an empty line right above this bullet?

Reply:
I think I might just ignore it and see how it publishes. If it continues to be a problem I'll make a bug ticket.